README.md

```diff
@@ -1,3 +1,122 @@
+
+### 1. CSS
+1. Copy the '@momentum-ui/core/fonts' directory to the 'fonts' directory for your site.
+2. Copy the '@momentum-ui/core/images' directory to the 'images' directory for your site.
+3. Copy the '@momentum-ui/core/css/momentum-ui.min.css' to your styles/css directory.
+4. In the `<head>`, of your HTML reference the location of your momentum-ui.min.css:
+
+    `<link rel='stylesheet' href='path/to/momentum-ui/css/momentum-ui.min.css'>`
+
+***
+### 2. SCSS
+1. Copy the '@momentum-ui/icons/fonts' directory to the 'fonts' directory for your site.
+2. Add an '$brand-font-folder, $icon-font-path variables to your app's variables.scss file.
+
+    ```scss
+    $brand-font-folder: "~@momentum-ui/core/fonts";
+    $icon-font-path: "~@momentum-ui/icons/fonts";
+    ```
+
+3. Import `@momentum-ui/core/scss/momentum-ui-components` into your main entry Scss file AFTER the variables.
+
+    ```scss
+    @import '~@momentum-ui/core/scss/momentum-ui-components';
+    ```
+
+4. Compile Sass files using your static compiler or bundler (Webpack, Grunt, Gulp, etc.)
+***
+
+Once Momentum styles have been imported into your project, you can start using the components. For example:
+
+  ```js
+  import {WebexAvatar} from '@webex/components';
+
+  function App() {
+    return (
+      <div className='App'>
+        <WebexAvatar title='Webex Avatar' />
+      </div>
+    );
+  }
+
+  export default App;
+  ```
+
+*Happy Coding!*
+
+## Contributing
+We'd love for you to contribute to our source code and to make Webex Components even better than it is today! Here are some [guidelines](https://github.com/webex/components/blob/master/CONTRIBUTING.md) that we'd like you to follow.
+
+### Issues
+Please open an [issue](https://github.com/webex/components/issues) and we will get to it in an orderly manner. Please leave as much as information as possible for a better underestanding.
+### Contributing Code
+We are using [Webex Style Guide eslint rule](https://github.com/webex/web-styleguide/tree/master/packages/node_modules/%40webex/eslint-config-react) and [prettier](https://github.com/prettier/prettier) to lint the code style.
+You can "prettify" your code before committing via:
+
+  ```bash
+  npm run prettier:write
+  ```
+
+### Release Process
+There is a list of commit types provided [here](https://github.com/webex/components/blob/master/CONTRIBUTING.md#type). However, not all commits triggers our release process. We are using [semantic-release](https://github.com/semantic-release/semantic-release) to fully automate the version management and package publishing. By default `semantic-release` uses the [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines) and triggers release and publishing based on the following rules:
+
+<center>
```

Comment:
What do this `<center>` tags do? The table it's not really being centered, and I think it's ok that way

Reply:
okay